pkgs/development/compilers/sbcl/default.nix

```diff
@@ -19,11 +19,6 @@
 
 let
   versionMap = {
-    # Only kept around for BCLM. Remove once unneeded there.
-    "2.1.9" = {
-      sha256 = "189gjqzdz10xh3ybiy4ch1r98bsmkcb4hpnrmggd4y2g5kqnyx4y";
-    };
-
     "2.4.2" = {
```

Comment:
love it. Can also set coreCompression to `true` and markRegionGC to `true` in default args.

Reply:
Wonderful news. I'll leave that to a future PR from someone that understands SBCL a little more. 🧷 

I'm just a curious lad looking for broken or odd packages.

Reply:
... I meant at the top of this file 😂